vcs_server/VcsPlot.py

```diff
@@ -4,6 +4,53 @@
 import sys
 import gc
 
+def fixValue(val):
+    if val == 100000000000000000000:
+        return 1e20
+    elif val == -100000000000000000000:
+        return -1e20
+    elif val == 100000002004087730000:
```

Comment:
Why do you have these additional special values?

Reply:
For whatever reason, that is one of the default level value set in `vcs/isoline.py`.

Reply:
@doutriaux1 Any idea why do we have this additional value? I think we should get rid of it unless there is a strong reason why we should keep it.